systemvm/patches/debian/config/opt/cloud/templates/check_heartbeat.sh.templ

```diff
@@ -47,13 +47,14 @@ then
     if [ $s -gt 2 ]
     then
         echo Keepalived process is dead! >> $ROUTER_LOG
-        $ROUTER_BIN_PATH/services.sh stop >> $ROUTER_LOG 2>&1
-        $ROUTER_BIN_PATH/disable_pubip.sh >> $ROUTER_LOG 2>&1
-        $ROUTER_BIN_PATH/primary-backup.sh fault >> $ROUTER_LOG 2>&1
         service keepalived stop >> $ROUTER_LOG 2>&1
         service conntrackd stop >> $ROUTER_LOG 2>&1
-	pkill -9 keepalived >> $ROUTER_LOG 2>&1
-	pkill -9 conntrackd >> $ROUTER_LOG 2>&1
+        
+        #Set fault so we have the same effect as a KeepaliveD fault.
+        python /opt/cloud/bin/master.py --fault
```

Comment:
will running the script here block this shell script until it exists, or will it launch as a daemon? If it's blocking then it's alright.